package.json

```diff
@@ -32,10 +32,8 @@
     "es2015/text.js"
   ],
   "scripts": {
-    "build": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
-    "clean": "rm -rf -- index.js text.js es2015",
-    "prepublish": "npm run clean && npm run build",
-    "test": "npm run build && mocha",
+    "build-patterns": "babel src -d .; NODE_ENV=es2015 babel src -d ./es2015; node script/inject-sequences.js",
```

Comment:
Okay cool. The only downside is that you might end up with a dirty working dir after the `npm install`. But the upside is that you automatically regenerate the files when you update the deps, and that it matches your other repos. I'll rename it back to build :)